pkg/utils/k8sutils.go

```diff
@@ -51,6 +51,8 @@ const (
 	// Value corresponding to annotation key has desired client hash
 	DesiredConfigHashAnnotationKey = "ocs.openshift.io/provider-side-state"
 
+	AnnotationNonResilientPoolsTopologyKey = "ocs.openshift.io/non-resilient-pools-topology-key"
```

Comment:
This annotation name does not fit the use case on the client side. We called the grpc topologyDomainLables so lets use a similar annotation name for this